distribution/assembly.xml

```diff
@@ -20,17 +20,17 @@
         <destName>source.war</destName>
       </file>
       <file>
-        <source>${project.basedir}/../opengrok-tools/${project.build.directory}/dist/opengrok-tools-${project.python.package.version}.tar.gz</source>
+        <source>${project.basedir}/../tools/${project.build.directory}/dist/opengrok-tools-${project.python.package.version}.tar.gz</source>
```

Comment:
so the target archive is still opengrok-tools? I'm all for, just asking.

Reply:
Yes, I think it should be recognizable among all the other Python packages. 'tools' would be too generic.